### ISSUE TITLE
Use cargo-binstall for cli

### ISSUE THREAD

**Issue by listepo** [Brendonovich/prisma-client-rust#455]
It would be great to make a `prisma-client-rust-cli` package via install [cargo-binstall](https://github.com/cargo-bins/cargo-binstall)

**Comment by Brendonovich** [comment]
From what I understand `cargo-binstall` and tools like it require publishing to crates.io, which I won't be doing (https://github.com/Brendonovich/prisma-client-rust/issues/76).
If it can support git dependencies then I'll consider it.

**Comment by listepo** [comment]
Maybe @NobodyXu can help

**Comment by NobodyXu** [comment]
Yes, cargo-binstall has `--git` support, as long as your repository have a `Cargo.toml`

**Comment by NobodyXu** [comment]
Checking the `Cargo.toml` for cli, it seems to be missing repository, and its version doesn't match the one is gh release, so if that"s updated, it might work out-of-the-box

**Comment by NobodyXu** [comment]
Checkout our doc for how cargo-binstall automatically discovers gh release artifacts https://github.com/cargo-bins/cargo-binstall/blob/main/SUPPORT.md#defaults

**Comment by listepo** [comment]
@NobodyXu thanks, @Brendonovich is this information useful to you?

**Comment by listepo** [comment]
@NobodyXu could you help with this task? since you have an understanding of what needs to be done

**Comment by NobodyXu** [comment]
Basically, you need to:

 - add repository for Cargo.toml of the cli crate
 - update the version of the for Cargo.toml of the cli crate to match release on github
 - update release pipeline to include pre-built artifacts

**Comment by listepo** [comment]
@Brendonovich for example https://github.com/trunk-rs/trunk/blob/main/.github/workflows/release.yaml
